ui/imports/shared/popups/walletconnect/DAppsListPopup.qml

```diff
@@ -87,113 +87,117 @@ Popup {
             model: root.model
             visible: !listPlaceholder.visible
 
-            delegate: DAppDelegate {
+            delegate: MouseArea {
```

Comment:
Why is the delegate no longer a `DAppDelegate`? I think the goal here should be to adjust that, not replace it with a random `MouseArea`

Reply:
DAppDelegate was an inline component and it's causing issues on qt 5.15.2. But I agree it should still be a DAppDelegate extracted in a file.

Reply:
In the scope of this PR I will move the delegates out to separate files, so there will be no crashes with Qt5.15.2.